ADR/ADR036-automate-patch-version-bumps.md

```diff
@@ -0,0 +1,79 @@
+# ADR036: Automate patch version bumps
+
+Date: 2024-10-31 :ghost:
+
+## Status
+
+Proposed
+
+## Context
+
+GOV.UK Forms uses [Dependabot](https://github.com/dependabot) to keep dependacies up to date. Dependancy update pull requests are raised automatically by Dependabot but a human has to approve and merge the pull requests. This can be an issue because:
+- we regularly have a large number of automated pull requests which require our attention. This means unnecessary toil, usually for the person on support
+- security patches can go unnoticed for longer than we would like and they are only applied when we take action
+
+A large number of PRs for a small team like that of GOV.UK Forms is not sustainable alongside other work, and being lax with
+dependency updates and their security fixes is not something we should be doing.
+
+### Semantice versioning
+In general, the packages consumed by GOV.UK Forms conform to [semantic versioning](https://semver.org/) rules[2], which
```

Comment:
Should this `[2]` still be here? It doesn't appear to be referenced anywhere else:

```suggestion
In general, the packages consumed by GOV.UK Forms conform to [semantic versioning](https://semver.org/) rules[2], which
```